koku/api/report/aws/query_handler.py

```diff
@@ -243,6 +247,16 @@ def execute_query(self):
                     account_alias=Coalesce(F(self._mapper.provider_map.get("alias")), "usage_account_id")
                 )
 
+                # Setup tags exist query for the returned data
+                tag_indicator_query = query_table.objects.filter(self.query_filter)
+                tag_indicator_query = tag_indicator_query.annotate(
+                    account_alias=Coalesce(F(self._mapper.provider_map.get("alias")), "usage_account_id"),
+                    tags_exist_sum=Sum(ExpressionWrapper(Coalesce(F('tags'), {}) != {}, output_field=IntegerField))
+                )
+                tag_indicator_query.group_by = ['account_alias']
```

Comment:
Using .values in the line below should handle the group by for you so you don't have to explicitly set it.